scripts/update-versions-for-release

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-# **Run this script and commit changes BEFORE creating a new release tag in Github**
+# **Run this script and commit changes AFTER creating a local tag for the new release and BEFORE creating a remote tag for the new release**
```

Comment:
Just confirming: creating local tag means tagging a drafted release (via GitHub UI)?

Reply:
creating a local tag = `git tag v1.0.2`  -- this is how we can break the cyclic dependency in the release process (commit with release version updates <-> tag push). 
With local tag creation, we can run our scripts like `update-versions-for-release` and get that commit merged in before creating the remote tag  and triggering a Travis build for release.  

creating a remote tag - this is expected to be done on Github UI 

I think the steps will be clear once I document the release process.